runtime/oak/src/xcm_config.rs

```diff
@@ -181,7 +181,7 @@ pub type Barrier = DenyThenTry<
 
 /// Based on the precedent set by other projects. This will need to be changed.
 pub fn dot_per_second() -> u128 {
-	CurrencyId::DOT.cent() * 16
+	CurrencyId::DOT.cent() * 16 // IRSAL: Laura's going to cover this with her PR
```

Comment:
her PR moves these values to a pallet, but we still need the right values for them